packages/js-config-webpack-plugin/test/fixtures/simple/.specific.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
maybe we shouldn’t put a .specific.babel.rc inside the simple fixture case but rather in a custom babel rc case - what do you think?

Reply:
Ok. It sounds good. I will create a new folder within the testing directory which is base of the babel rc case.